packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding_phone_confirmation/blocs/onboarding_phone_bloc.dart

```diff
@@ -136,9 +136,9 @@ class OnboardingPhoneBloc extends $OnboardingPhoneBloc {
           .whereSuccess()
           .doOnData((_) {
         if (_isOnboarding) {
-          _router.go(const OnboardingPhoneConfirmRoute().routeLocation);
+          _router.push(const OnboardingPhoneConfirmRoute().routeLocation);
```

Comment:
Please prefer using the go method instead of the push

Reply:
Restored previous version. ✔️